docs/apis-tools/tasklist-api-rest/tasklist-api-rest-authentication.md

```diff
@@ -5,63 +5,137 @@ sidebar_position: 2
 description: "Describes authentication options that can be used to access Tasklist REST API."
 ---
 
-## Authentication in the cloud
-
-To access the API endpoint, you need an access token.
-
-Your client must send a header in each request:
-
-`Authorization: Bearer <Token>`
-
-For example, send a request using _curl_:
-
-```shell
-curl -X POST -H -H :accept: application/json" -H "Authorization: Bearer <TOKEN>" -d '' http://localhost:8080/v1/tasks/search
-```
-
-### How to obtain the access token
-
-You must obtain a token to use the Tasklist API. When you create a Tasklist [client](/guides/setup-client-connection-credentials.md), you get all the information needed to connect to Tasklist.
-
-Refer to our guide on [building your own client](../build-your-own-client.md).
-
-The following settings are needed:
-
-| Name                     | Description                                     | Default value         |
-| ------------------------ | ----------------------------------------------- | --------------------- |
-| client id                | Name of your registered client                  | -                     |
-| client secret            | Password for your registered client             | -                     |
-| audience                 | Permission name; if not given use default value | `tasklist.camunda.io` |
-| authorization server url | Token issuer server                             | -                     |
-
-Send a token issue _POST_ request to the authorization server with the following content:
-
-```json
-{
-  "client_id": "<client-id>",
-  "client_secret": "<client-secret>",
-  "audience": "<audience>",
-  "grant_type": "client_credentials"
-}
-```
-
-Refer to the following example with _curl_:
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+All Tasklist API requests require authentication. To authenticate, generate a JWT token and pass it in each request.
+
+## Generating a token
+
+<Tabs groupId="authentication" defaultValue="saas" queryString values={
```

Comment:
Bit of a nit pick, but could you [sync](https://docusaurus.io/docs/2.x/markdown-features/tabs#syncing-tab-choices) the tabs so they are both on SaaS or both on SM, not a combination? This requires the same `groupId` prop.

Reply:
Definitely, this is a good idea, and I misunderstood the `groupId` property. I remember thinking "oh I bet this has to be unique on the page, there's no way docusaurus will sync these tabs across different sets, I'd better use different groupIds or it's going to cause some weird behavior/an error." I should have more faith 😅